src/plugins/data/public/search/search_interceptor.ts

```diff
@@ -106,12 +114,13 @@ export class SearchInterceptor {
         return new EsError(e);
       }
     } else {
-      return e;
+      return new Error(e.message);
```

Comment:
should we check `e instanceof Error ? e : new Error(e.message);`
it could be that something inside the search returned an error we didn't expect (e.g. regular run time error/bug).
Would be nice to not lose the stracktrace